p2p/src/sync/tests/block_response.rs

```diff
@@ -112,23 +139,34 @@ where
 
 #[tokio::test]
 async fn valid_block_invalid_state_libp2p() {
-    valid_block_invalid_state::<Libp2pService>(make_libp2p_addr(), PeerId::random()).await;
+    valid_block_invalid_state::<MakeP2pAddress, Libp2pService>(PeerId::random()).await;
 }
 
 // TODO: fix https://github.com/mintlayer/mintlayer-core/issues/375
 #[tokio::test]
 #[cfg(not(target_os = "macos"))]
-async fn valid_block_invalid_state_mock() {
-    valid_block_invalid_state::<MockService>(make_mock_addr(), MockPeerId::random()).await;
+async fn valid_block_invalid_state_mock_tcp() {
+    valid_block_invalid_state::<MakeTcpAddress, MockService<TcpMockTransport>>(MockPeerId::random()).await;
```

Comment:
Please make sure that all tests use the same parameters but only differ by generic arguments.